src/Components/authenticate/SignOut.jsx

```diff
@@ -0,0 +1,57 @@
+import React, {useState,useContext,Fragment} from "react";
+import {Helmet} from "react-helmet";
+import { Link } from "@reach/router";
+import { signOut } from "../../firebase";
+import SignIn from "../authenticate/SignIn"
+import { useHistory } from "react-router"
+
+const SignOut = () => {
+    const [error, setError] = useState(null);
+    const history = useHistory();
+    let user = 'N/A';
+    if(localStorage.getItem('User') == null) {
+        localStorage.setItem('User', 'N/A');
+    }
+    else {
+        user = localStorage.getItem('User');
+    }
+
+    const handleSignOut = () => {
+        signOut();
+        history.push({
+            pathname:  "/",
+        });
+    }
+    const handleHome = () =>{
+        history.push({
+            pathname:  "/",
+        });
+    }
+
+    return (
+    (user != 'N/A')  

```

Comment:
same question as signIn, why not just leave it as null?

Reply:
Read above.